### PR TITLE
Junit xml finishline

### PR DESCRIPTION
**Pull request by minusworld**

### CODE REVIEW

**Comment by minusworld** [comment]
Supercedes https://github.com/returntocorp/semgrep/pull/1495.
Closes https://github.com/returntocorp/semgrep/issues/1388

**Comment by minusworld** [comment]
@nightwatchcyber I'm attempting to get this PR over the finish line. I saw some discussion on the previous one that talked about a `--fmt` flag but to my knowledge it hasn't been implemented. This one uses the original `--junit-xml` flag.

**Comment by nightwatchcyber** [comment]
> @nightwatchcyber I'm attempting to get this PR over the finish line. I saw some discussion on the previous one that talked about a `--fmt` flag but to my knowledge it hasn't been implemented. This one uses the original `--junit-xml` flag.

Thank you!

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (brendongo) [inline Pipfile]: Probably unintended file

File: semgrep/Pipfile [hunk 2]
```diff
@@ -10,9 +10,11 @@ tox = "*"
 pytest-snapshot = "*"
 appdirs = "*"
 pytest-xdist = "*"
+xmldiff = "*"
 
 [packages]
 semgrep = {editable = true,path = "."}
+pytest = "*"
```
Comment (brendongo) [inline semgrep/Pipfile]: Unintended add?

File: semgrep/setup.py [hunk 3]
```diff
@@ -122,11 +122,13 @@ def run(self):
     long_description_content_type="text/markdown",
     url="https://github.com/returntocorp/semgrep",
     install_requires=[
+        "attrs>=19.3.0",
         "colorama>=0.4.3",
+        "junit_xml==1.9",
+        "packaging>=20.4",
```
Comment (brendongo) [inline semgrep/setup.py]: This is on here twice

File: Pipfile (hunk 1 above)
Comment (minusworld) [reply Pipfile]: Is there not a Pipfile in the top level of the repo? It was there when I inherited it but assumed it was from ours.

**Review by brendongo — APPROVED** [review]

**Pull request merged**